lib/reform/form/multi_parameter_attributes.rb

```diff
@@ -24,10 +24,12 @@ def call(params)
 
   private
     def params_to_date(year, month, day, hour, minute)
-      return nil if [year, month, day].any?(&:blank?)
+      if [year, month, day].any?(&:blank?) || !Date.valid_date?(*[year, month, day].map(&:to_i))
```

Comment:
I think about it, I just try solve the problem with minimum code, but I will do it, above have the same array too.